scalableminds/webknossos
# Fix “merging” a single volume annotation

Merging multiple annotations is implemented by `reduceLeft(mergeTwo)` – so if only one goes in, it is returned unchanged.
This means, the version reset was not executed in this case, which lead to errors for volume annotations.

### URL of deployed dev instance (used for testing):
- https://fixmergevolumes.webknossos.xyz

### Steps to test:
- Create volume tasktype, project and single-instance task. request it, trace and finish
- As admin, click view in task list (not individual user instance), should see volume data


### Issues:
- fixes #5197

------
